x-pack/test/fleet_api_integration/config.base.ts

```diff
@@ -76,6 +76,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
           'agentTamperProtectionEnabled',
           'enableStrictKQLValidation',
           'subfeaturePrivileges',
+          'enablePackagesStateMachine',
```

Comment:
I enabled the feature flag on the integration teststo verify that everything works as expected. I can disable it then if we don't want to keep it.